nrf70_zephyr_shim/source/os/shim.c

```diff
@@ -26,7 +26,7 @@
 #include "osal_ops.h"
 #include "qspi_if.h"
 
-LOG_MODULE_REGISTER(wifi_nrf, CONFIG_WIFI_NRF700X_SHIM_BUS_LOG_LEVEL);
+LOG_MODULE_REGISTER(wifi_nrf, CONFIG_WIFI_NRF700X_SHIM_LOG_LEVEL);
```

Comment:
I guess this is not related to the reg-domain API extension?

Reply:
No, general improvement, please see the commit log for details.

Reply:
ok